solr/core/src/java/org/apache/solr/core/ZkContainer.java

```diff
@@ -182,6 +188,15 @@ public List<CoreDescriptor> getCurrentDescriptors() {
         if(boostrapConf) {
           ZkController.bootstrapConf(zkController.getZkClient(), cc);
         }
+        MetricsMap metricsMap = new MetricsMap(zkController.getZkClient().getMetrics());
+        metricProducer = new SolrMetricProducer() {
+          @Override
+          public void initializeMetrics(SolrMetricsContext parentContext, String scope) {
+            SolrMetricsContext ctx = parentContext.getChildContext(this);
+            ctx.gauge(null, metricsMap,
+                true, scope, "", SolrInfoBean.Category.CONTAINER.toString());
```

Comment:
The first `""` arg to the `String... metricPath` arg has no effect, right? i.e. it gets ignored [here](https://github.com/fullstorydev/lucene-solr/blob/bca19765321c9be82e5268bde657cc676b2549ff/solr/core/src/java/org/apache/solr/metrics/SolrMetricManager.java#L849-L851)?
If so, then it could be removed.